recipe/bld.bat

```diff
@@ -8,7 +8,7 @@ cmake ^
     -DCMAKE_INSTALL_PREFIX=%LIBRARY_PREFIX% ^
     -DCMAKE_BUILD_TYPE=Release ^
     -DOSG_TEXT_USE_FONTCONFIG=OFF ^
-    -DCOLLADA_DOM_DIR=%PREFIX%\include\collada-dom2.5 ^
+    -DCOLLADA_DOM_ROOT=%PREFIX% ^
```

Comment:
Not %LIBRARY_PREFIX% in this case?

Reply:
Yes, explaining certainly why it is not passing yet. Thanks @wolfv for the suggestion.

Reply:
On Windows, it is still not able to find COLLADA